mbld/deps.myr

```diff
@@ -461,7 +463,7 @@ const linkcmd = {b, n, mt, bin, libs, dynlibs, istest
 	if std.sleq(opt_sys, "osx")
 		std.slpush(&n.cmd, std.sldup("-macosx_version_min"))
 		std.slpush(&n.cmd, std.sldup("10.6"))
-	elif std.sleq(opt_sys, "linux") && b.dynlink
+	elif std.sleq(opt_sys, "linux") && dynlink
```

Comment:
Almost :)

    b.dynlink || dynlink

the libraries *or* the binary can add a dynamic library dependency.